lib/index.js

```diff
@@ -82,14 +82,19 @@ module.exports = function(options) {
       }
     }
 
-    var payload = jwt.decode(token);
+    var dtoken = jwt.decode(token, { complete: true }) || {};
```

Comment:
Null is a possible return value. I defaulted it so I could safely chain off later when passing dtoken.header and dtoken.payload to the callback

Sent from my iPhone

> On May 15, 2015, at 12:19 PM, Damian Schenkelman <notifications@github.com> wrote:
> 
> In lib/index.js:
> 
> > @@ -82,14 +82,19 @@ module.exports = function(options) {
> >        }
> >      }
> >  
> > -    var payload = jwt.decode(token);
> > +    var dtoken = jwt.decode(token, { complete: true }) || {};
> was {} possible before? what should happen if {}?
> 
> —
> Reply to this email directly or view it on GitHub.
>